db/migrate/20170601123221_create_attestations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAttestations < ActiveRecord::Migration[5.0]
+  def change
+    create_table :attestations do |t|
+      t.string :pdf
+      t.string :title
```

Comment:
Si on est sur que c'est utile oui, sinon less code is better, et ça simplifie l'UI au passage. Mais si c'est dans notre spécification y'a aucun problème.